Models/SSBTaxAssessment/GraphQLTypes/TaxAssessmentDatasetType.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Backend.Models.Base.JsonStat;
+using Backend.Models.SSBTaxAssessment.POCO;
+using Backend.utils;
+using GraphQL;
+using GraphQL.Types;
+
+namespace Backend.Models.SSBTaxAssessment.GraphQLTypes
+{
+    public class
+        TaxAssessmentDatasetType : JsonStatDatasetType<TaxAssessment.TaxAssessmentDimension, TaxAssessmentDimensionType>
+    {
+        public TaxAssessmentDatasetType() : base(true)
+        {
+            Field<ListGraphType<LabeledValueTaxType>>("value", "The values for the selected municipalities and years.",
+                null,
+                context =>
+                {
+                    var municipalities = context.Parent.GetArgument<List<string>>("municipalities");
+                    var years = context.Parent.GetArgument<List<string>>("years");
+                    var skipSize = context.Source.Dimension.Size[3];
+                    var municipalityEntrySize = context.Source.Dimension.Size[0];
+                    return (from municipality in municipalities
+                        let municipalityYears = context.Source.Value
+                            .Skip(NorwayTools.MunicipalityCodeToIndexTaxes[municipality] * 294)
+                            .Take(municipalityEntrySize)
+                            .ToList()
+                        select new LabeledValueTax
+                        {
+                            Municipality = NorwayTools.MunicipalityCodeToMunicipalityNameTaxes[municipality],
+                            TaxesForYear = new Collection<TaxesForAGivenYear>(years.Select(year =>
+                                    new TaxesForAGivenYear
+                                    {
+                                        Year = year,
+                                        Brutto = municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize],
```

Comment:
Of this;

```c#
municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize + n],
```

Only `n` changes for each line. Extract `NorwayTools.YearToIndexTaxes[year]` to a variable in the loop, and perhaps create a lambda function for retrieving the `n`th offset variable.

Reply:
Even though we decided not to split up the function into smaller pieces, I believe this should still be implemented to make the code a bit DRY-er.

Reply:
Added now